instrumentation/opentelemetry_cowboy/test/opentelemetry_cowboy_SUITE.erl

```diff
@@ -9,8 +9,6 @@
 -include_lib("opentelemetry/include/otel_span.hrl").
 -include_lib("opentelemetry_api/include/otel_tracer.hrl").
 
--define(assertListsMatch(List1, List2), ?assertEqual(lists:sort(List1), lists:sort(List2))).
```

Comment:
remove this as we it's not used anymore. we are using `assertMatch` to assert the attributes instead